notification.c

```diff
@@ -133,6 +135,17 @@ struct mako_notification *get_notification(struct mako_state *state,
 	return NULL;
 }
 
+struct mako_notification *get_tagged_notification(struct mako_state *state,
+		const char *tag, const char *app_name) {
+	struct mako_notification *notif;
+	wl_list_for_each(notif, &state->notifications, link) {
+		if (strcmp(notif->tag, tag) == 0 && strcmp(notif->app_name, app_name) == 0) {
```

Comment:
Just noticed a glaring issue with this check: notifications that don't provide any tag at all are going to match each other. Will open a fix.